overlord/devicestate/handlers_install.go

```diff
@@ -1010,6 +1012,30 @@ func (m *DeviceManager) doInstallFinish(t *state.Task, _ *tomb.Tomb) error {
 	}
 	defer unmountParts()
 
+	if !sys.Model.Classic() {
+		sysSeed, err := seedOpen(dirs.SnapSeedDir, sys.Label)
+		if err != nil {
+			return fmt.Errorf("cannot open seed: %w", err)
+		}
+
+		if copier, ok := sysSeed.(seed.Copier); ok {
+			db := assertstate.DB(st)
+			err = copier.LoadAssertions(db, func(b *asserts.Batch) error {
+				return assertstate.AddBatch(st, b, nil)
+			})
+			if err != nil {
+				return fmt.Errorf("cannot load seed assertions: %w", err)
+			}
+
+			logger.Debugf("copying label %q to seed partition", sys.Label)
+			if err := copier.Copy(seedMntDir, sys.Label, timings.New(nil)); err != nil {
```

Comment:
we have infrastructure to measure timing for tasks, maybe it's time to hook that up here